backend/clubs/serializers.py

```diff
@@ -1285,6 +1285,14 @@ def create(self, validated_data):
         # New clubs created through the API must always be approved.
         validated_data["approved"] = None
 
+        request = self.context.get("request", None)
+        perms = request and request.user.has_perm("clubs.approve_club")
+
+        if not perms and (not settings.QUEUE_OPEN or not settings.NEW_QUEUE_OPEN):
```

Comment:
Just to be clear: this is an AND because admins can always create approved clubs, right?